docs/releases/v1.15.5-4.md

```diff
@@ -0,0 +1,26 @@
+# Release notes
+
+- [Release notes](#release-notes)
+  - [Route table refactoring](#route-table-refactoring)
+  
+This version refactors how we manage route table rules on private and public route tables. The need is to remove the rules list from the aws_route_table terraform resource and to permit external route table edits (for example: adding a vpc peering connection).
+
+
+## Route Table Refactoring
+
+### Issue
+
+Cannot customize route table rules with external terraform projects
+
+### Change
+
+Removed route object from aws_route_table public and private. Created the corresponding resources aws_route public and private.
+
+#### How to upgrade provider
+
+The upgrade path needs manual intervention on the AWS side.
+The old egress route entries must be removed from the current route tables (public and private).
+
+Then, you can apply the new configuration.
+
+Downtime is expected if the services running inside the cluster need internet connectivity.
```

Comment:
Can you better quantify downtime?

Furthermore, do you think that in order to reduce the downtime we can use `terraform apply -target=...` in order not to process the whole Terraform state?

Reply:
That's exactly what i was looking for ;)

Reply:
Added a little more information on upgrade path!

Reply:
awesome, thanks!